build-tools.php

```diff
@@ -245,7 +245,9 @@ protected function buildLogEntry(array $pr): string
         $changeLogEntry = ["`{$pr['title']} <{$pr['html_url']}>`_"];
         $this->addHeadlineMarkup($changeLogEntry, '-');
         $changeLogEntry[] = '';
-        $changeLogEntry[] = $this->cleanupPrMessage($pr['body']);
+        if (isset($pr['body'])) {
```

Comment:
You might need to check if it's a string, since it tell that "null" is given - so the key could be considered set. Or cast directly to string, since (string) null = ''

Reply:
Good points ... will update that later :)